cli/cmd/start_cmd.go

```diff
@@ -4,6 +4,7 @@ import (
 	"context"
 	"os"
 
+	"github.com/davecgh/go-spew/spew"
```

Comment:
Should we keep `spew`? I noticed that we tried to remove it on [this commit](https://github.com/kubeshop/tracetest/commit/7d6bf5f6fb0424bb4ca23c7aa496416e45259282).

Reply:
ah damn, good catch. fixed on main